src/main/java/com/linecorp/armeria/client/ClientBuilder.java

```diff
@@ -46,12 +48,18 @@
 
     private RemoteInvokerFactory remoteInvokerFactory = RemoteInvokerFactory.DEFAULT;
     private Function<Client, Client> decorator;
+    private String groupName;
```

Comment:
- could be `final`
- could be moved right after `uri` for meaningful field grouping